internal/commands/build/build.go

```diff
@@ -169,7 +169,9 @@ func buildImageUsingBuildx(app *types.App, contextPath string, opt buildOptions,
 		out = os.NewFile(dockerCli.Out().FD(), "/dev/stdout")
 	}
 
-	pw := progress.NewPrinter(ctx, out, opt.progress)
+	ctx2, cancel := context.WithCancel(context.TODO())
```

Comment:
I think a regression test could be tricky to add here, but maybe a one line comment explaining why we have another context?

Reply:
maybe we can find a better name, like `printerContext`?
```suggestion
	printerContext, cancel := context.WithCancel(context.TODO())
```